auth-server/src/model/Notification.ts

```diff
@@ -3,48 +3,40 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import { Model } from 'objection';
+
+import { JsonSchema } from '../types';
 import connection from './connection';
 
 Model.knex(connection);
 
 export default class Notification extends Model {
 	readonly id!: number
+	new_proposal!: boolean
+	own_proposal!: boolean
+	post_created!: boolean
+	post_participated!: boolean
 	user_id!: number
-	post_participated: boolean
-	post_created: boolean
-	new_proposal: boolean
-	own_proposal: boolean
 
-	static get tableName () {
+	static get tableName (): string {
 		return 'notification';
 	}
 
-	static get idColumn () {
+	static get idColumn (): string {
```

Comment:
```suggestion
	static get idColumn (): number {
```

Reply:
return value is string

Reply:
But it shouldn't, id s an id, it's specified here a couple lines above: https://github.com/paritytech/polkassembly/pull/613/files/096fdc0d6e5d73a8de1faac54ef586b8ad5bbdf6#diff-929aa84a176da7a6f0d11b79e8770b49R13

Reply:
id type is a number yes. That is mentioned on line 13. idColumn return type is  string

Reply:
if you change it to number the error is 
Type '"id"' is not assignable to type 'number'.ts(2322)

Reply:
ok, now I understand what that idColumn is.. it's the idColumn**Name**. Then I realized that it's never used.